THANKS.txt

```diff
@@ -227,6 +227,7 @@ Leo P. Singer for bug fixes in scipy.optimize and scipy.interpolate and
 Domen Gorjup and Janko Slavič for continuous wavelet transform with complex
     wavelets fix.
 Søren Fuglede Jørgensen for improvements to scipy.sparse.csgraph
+Grzegorz Mrukwa for a bug fix in rectangular_lsap.cpp
```

Comment:
Not sure that this should be in this PR?

Reply:
It seems that I accidentally took THANKS.txt from the master branch. Here I removed the line, but doing so causes conflicts. Is it OK? or maybe my git operation is wrong?